script/generate-docs

```diff
@@ -19,4 +19,4 @@ STDERR.puts "Generating docs against #{remote}/#{branch}."
 # Explicitly use the guides/tsconfig.json file, because it includes
 # both the source and docs. The build itself just uses the root tsconfig,
 # which just hits src.
-`node node_modules/typedoc/bin/typedoc --tsconfig guides/tsconfig.json --options config/typedoc.json --gitRemote #{remote} --gitRevision #{branch}`
+`node node_modules/typedoc/bin/typedoc --plugin typedoc-plugin-merge-modules --tsconfig guides/tsconfig.json --gitRemote #{remote} --gitRevision #{branch}`
```

Comment:
Just confirming using this plugin the URLs do not change right?

Reply:
Yes, I mentioned in the PR description